### ISSUE TITLE
collect useful links for (usp) students

### ISSUE THREAD

**Issue by SwampertX** [uschongensec/cinnabot#6]
including 
- [ ] [faulty cinnamon](https://bit.ly/faultycinnamon)
- [ ] [fund request](https://gg.gg/fundreq)
telegram groups such as 
- [ ] battle royale
- [ ] royal bistro
- [ ] supper jio
- [ ] ig groups
- [ ] uschannel
- [ ] house channels

**Comment by SwampertX** [comment]
Currently such links are kept under the `/resources` command. We could further extend it by adding the above in

**Comment by suwoons** [comment]
Working on adding the IG list under resources
Might consider sorting them into categories (eg. "Groups and Channels", "Websites", "Interest Groups") and adding buttons for them, instead of displaying all the info at once like how it currently is

![image](https://user-images.githubusercontent.com/50527491/61179423-743a5180-a634-11e9-8375-0d8d1d8b60a3.png)

**Comment by chloelee767** [comment]
The USC website has a list of IGs (https://nususc.com/groups) which is stored in firestore - so you can read the IG list from firestore, similar to spaces function.

**Comment by SwampertX** [comment]
The thing is this - the IG list on USC website is dependent on each IG's in-charge to update every year. A lot of the data on the USC website is outdated. In fact, the information disseminated to the freshies (the master IG list) every year would be the most up-to-date source.

That being said, I support that we should only have a Single Source of Truth that lies within the USC website (and it is login protected, although weak). This would then require the IG coordinators to change their workflows. Then on the freshies can just visit the website/cinnabot to get consistent information.

*TLDR:* i think we should read from firestore, but need to cooperate with the VP (Comm Life) or IG Coordinator to change the IGs' workflow. My suggestion we could first hardcode the IG links in for now, but also code, in another branch, the method using the firebase data, swapping them as soon as the new IG registration workflow kicks in.
